op-chain-ops/deployer/init.go

```diff
@@ -52,30 +67,44 @@ func Init(cfg InitConfig) error {
 	intent := &state.Intent{
 		L1ChainID:       cfg.L1ChainID,
 		UseFaultProofs:  true,
-		FundDevAccounts: cfg.Dev,
+		FundDevAccounts: true,
```

Comment:
Do you plan to later derive the `UseFaultProofs` and `FundDevAccounts` values from input configs or will they always be `true` in this `Init()` function?

Reply:
the `Init()` function is used to generate a default config, so it'll likely always be true. users won't use the default values in production.